index.js

```diff
@@ -51,4 +52,17 @@ if (Platform.OS === 'android') {
         () => androidBgMessagingHandler
```

Comment:
since you removed the import, i think you have to remove the function call also.